runtime/themes/term16_light.toml

```diff
@@ -0,0 +1,85 @@
+# Author: dgkf <doug.kelkhoff@gmail.com>
+# Modified from base16_terminal, Author: NNB <nnbnh@protonmail.com>
+
+inherits = "term16-dark"
```

Comment:
The name here should be `term16_dark`. Inherit is not working.

Reply:
Ah good catch - that was a late change when I realized the more common convention is to use underscores. Will make a quick fix.